src/stores/bots.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { defineStore } from 'pinia'
+import Vue from 'vue'
+
+import { BOT } from '../constants.js'
+import { disableBotForConversation, enableBotForConversation, listBotsForConversation } from '../services/botsService.js'
+
+export const useBotsStore = defineStore('bots', {
+	state: () => ({
+		bots: {},
+	}),
+
+	getters: {
+		getBots: (state) => (token) => {
+			if (state.bots[token]) {
+				return Object.values(state.bots[token])
+			} else {
+				return []
+			}
+		},
+	},
+
+	actions: {
+		/**
+		 * Update information about available bots for conversation
+		 *
+		 * @param {string} token The conversation token
+		 */
+		async updateBotsInformation(token) {
+			if (!this.bots[token]) {
+				Vue.set(this.bots, token, {})
+			}
+
+			const response = await listBotsForConversation(token)
+
+			response.data.ocs.data.forEach((bot) => {
+				Vue.set(this.bots[token], bot.id, bot)
+			})
+		},
```

Comment:
**thoughts:** I'm not sure about "information" in naming here.

Reply:
`loadConversationBots` should be here, I suppose now